go.mod

```diff
@@ -1,66 +1,72 @@
 module k8s.io/cloud-provider-gcp
 
 require (
-	k8s.io/api v0.19.0
-	k8s.io/apimachinery v0.19.0
-	k8s.io/apiserver v0.19.0
-	k8s.io/client-go v0.19.0
-	k8s.io/cloud-provider v0.19.0
-	k8s.io/component-base v0.19.0
-	k8s.io/kube-controller-manager v0.19.0
-	k8s.io/kubernetes v1.19.0
-	k8s.io/legacy-cloud-providers v0.19.0
+	k8s.io/api v0.20.0
+	k8s.io/apimachinery v0.20.0
+	k8s.io/apiserver v0.20.0
+	k8s.io/client-go v9.0.0+incompatible
```

Comment:
cc @caesarxuchao but this client-go version seems wrong.

Reply:
v0.20.0 looks congruent? https://github.com/kubernetes/client-go/releases/tag/v0.20.0

Reply:
I assume Line 12 `k8s.io/client-go => k8s.io/client-go v0.20.0` will get the right version for client go?

Reply:
Yes, the `replace` will result in the correct version. It would be nice to have all the `require` statements aligned on `v0.20.0`, just to minimize confusion. Does something like `go get k8s.io/client@v0.20.0` will update that line appropriately?

Reply:
Updated go.mod in the latest commit.